### PR TITLE
Consuming recipes from ConanCenter

### PR DESCRIPTION
**Pull request by prince-chrismc**

### CODE REVIEW

File: _posts/2022-09-01-consuming-recipes-during-migration-conancenter.markdown [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+ "publisher": {
+    "@type": "Organization",
+    "name": "Conan.io",
+    "logo": {
+      "@type": "ImageObject",
+      "url": "https://media.jfrog.com/wp-content/uploads/2017/07/20134853/conan-logo-text.svg"
+    }
+},
+ "datePublished": "2022-08-21",
+ "description": "Improved download tool to support getting files from the local file system, support for components in MSBuildDeps, new MesonDeps generator, improved CMakePresets integration and lots of fixes to ease Conan 2.0 migration.",
```
Comment (czoido) [inline _posts/2022-09-01-consuming-recipes-during-migration-conancenter.markdown:27]: This part is still here from another blogpost

**Review by czoido — APPROVED** [review]

**Comment by uilianries** [comment]
Current view:

Screenshot 2022-09-19 at 11-29-05 ConanCenter is rolling out Conan 2 0 recipes - Are you ready

**Review by uilianries — APPROVED** [review]

File: _posts/2022-09-01-consuming-recipes-during-migration-conancenter.markdown [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+ "keywords": "c c++ package manager conan release", 
+ "publisher": {
+    "@type": "Organization",
+    "name": "Conan.io",
+    "logo": {
+      "@type": "ImageObject",
+      "url": "https://media.jfrog.com/wp-content/uploads/2017/07/20134853/conan-logo-text.svg"
+    }
+},
+ "datePublished": "2022-08-21",
```
Comment (prince-chrismc) [inline _posts/2022-09-01-consuming-recipes-during-migration-conancenter.markdown:26]:
```suggestion
 "datePublished": "2022-09-01",
```

File: _posts/2022-09-01-consuming-recipes-during-migration-conancenter.markdown [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+title: "ConanCenter is rolling out Conan 2.0 recipes - Are you ready?"
+meta_title: "ConanCenter is rolling out Conan 2.0 recipes - Are you ready?" 
+meta_description: "Learn how to safely consumer recipes using your very own ArtifactoryCE"
+---
+
+<script type="application/ld+json">
+{ "@context": "https://schema.org", 
+ "@type": "TechArticle",
+ "headline": "ConanCenter is rolling out Conan 2.0 recipes - Are you ready?d",
+ "alternativeHeadline": "Learn how to setup your enviroment to consumer recipes from ConanCenter",
```
Comment (prince-chrismc) [inline _posts/2022-09-01-consuming-recipes-during-migration-conancenter.markdown:13]:
```suggestion
 "alternativeHeadline": "Learn how to setup your environment to consume recipes from ConanCenter",
```

File: _posts/2022-09-01-consuming-recipes-during-migration-conancenter.markdown [hunk 4]
```diff
@@ -0,0 +1,83 @@
+---
+layout: post
+comments: false
+title: "ConanCenter is rolling out Conan 2.0 recipes - Are you ready?"
+meta_title: "ConanCenter is rolling out Conan 2.0 recipes - Are you ready?" 
+meta_description: "Learn how to safely consumer recipes using your very own ArtifactoryCE"
```
Comment (prince-chrismc) [inline _posts/2022-09-01-consuming-recipes-during-migration-conancenter.markdown:6]:
```suggestion
meta_description: "Learn how to safely consume public recipes using your very own ArtifactoryCE"
```

File: _posts/2022-09-01-consuming-recipes-during-migration-conancenter.markdown (hunk 1 above)
Comment (prince-chrismc) [reply _posts/2022-09-01-consuming-recipes-during-migration-conancenter.markdown:27]:
```suggestion
 "datePublished": "2022-08-21",
 "description": "Learn how to safely consume recipes from ConanCenter using your very own ArtifactoryCE.",
```

**Review by prince-chrismc — COMMENTED** [review]
Spelling

File: _posts/2022-09-01-consuming-recipes-during-migration-conancenter.markdown [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+ "publisher": {
+    "@type": "Organization",
+    "name": "Conan.io",
+    "logo": {
+      "@type": "ImageObject",
+      "url": "https://media.jfrog.com/wp-content/uploads/2017/07/20134853/conan-logo-text.svg"
+    }
+},
+ "datePublished": "2022-09-01",
+ "description": "Improved download tool to support getting files from the local file system, support for components in MSBuildDeps, new MesonDeps generator, improved CMakePresets integration and lots of fixes to ease Conan 2.0 migration.",
```
Comment (prince-chrismc) [inline _posts/2022-09-01-consuming-recipes-during-migration-conancenter.markdown:27]:
```suggestion
 "description": "Learn how to safely consume recipes from ConanCenter using your very own ArtifactoryCE.",
```

**Comment by prince-chrismc** [comment]
Sorry for the spam, GitHub was not cooperating with batch suggestions 🤷

**Review by memsharded — APPROVED** [review]

**Pull request merged**